src/appengine/scripts/redirector.py

```diff
@@ -258,6 +258,12 @@ def trailing_slash(handler, *args, **kwargs):
       defaults={'_uri': '/community/'}),
     Route('/articles/feet-wet-streams/', RedirectHandler,
       defaults={'_uri': '/docs/tutorials/streams/'}),
+    Route('/articles/dart-unit-tests/', RedirectHandler,
+      defaults={'_uri': 'https://pub.dartlang.org/packages/test'}),
+    Route('/articles/mocking-with-dart/', RedirectHandler,
+      defaults={'_uri': 'https://pub.dartlang.org/packages/test'}),
```

Comment:
The test package doesn't provide support for mocking. We're now recommending that people use [mockito](https://pub.dartlang.org/packages/mockito), so linking to that is probably best.